### ISSUE TITLE
Bug (Keyboard Accessibility): Fix focus style of secondary button in dark mode

### ISSUE THREAD

**Issue by dsmchen** [TheOdinProject/theodinproject#5202]
### Checks

- [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [x] The title of this issue follows the `Bug: brief description of bug` format, e.g. `Bug: Lesson complete button does not update on click`
- [x] Would you like to work on this issue?

### Bug description

- Parent issue: https://github.com/TheOdinProject/theodinproject/issues/2893
- Currently, focus style colour contrast ratio is approx 2:1. It should be at least 3:1.

### How to reproduce

1. Go to https://www.theodinproject.com/
2. Select dark mode
3. Tab to "View Full Curriculum"
4. Focus style is difficult to perceive

### Expected behavior

1. Go to https://www.theodinproject.com/
2. Select dark mode
3. Tab to "View Full Curriculum"
4. Focus style is easy to perceive

### What browsers are you seeing the problem on?

Firefox, Chrome

### What OS are you using?

MacOS

### Discord Name

doris6601
